CHANGELOG.markdown

```diff
@@ -1,3 +1,7 @@
+# 0.2.1
```

Comment:
- Usually when doing a PR to a gem/library, just put something like `NEXT`, and leave it to the maintainers to bump the version..
- Also to follow semvar, this is a new "feature", so I probably should bump it to `0.3.0`... but it's such a small feature I'll .z release it

So no worries this time, just a note for the next PRs you send 😄

Reply:
Awesome, good to know. Thank you for the feedback!